### ISSUE TITLE
Embedded storage question

### ISSUE THREAD

**Issue by MeteorBoom** [RefreshingIO/hangfire-ravendb#18]
Why GlobalConfiguration.Configuration.UseEmbeddedRavenStorage() option is removed since 2.0 version? Is new way exist to configure it? In README this way is stayed. I am confused a little...

**Comment by jblacker** [comment]
I did some digging and found that they were removed in commit 5d27d05ac0e73d7c94e3d3c39bea39e2eff78600, but it's unclear why.  This might have been an oversight or there was some undocumented reason for it.  I hope that @AustinWinstanley  or the committer @derekdhaas  has an answer though.  I'm going to fork and see if I can add it back in and and find out why they might have been removed when I have some time.

**Comment by jblacker** [comment]
@MeteorBoom Ok, so it looks like this was done because the Embedded version doesn't currently work with .NET Core.  They would need to do conditional compilation to restore the embedded version functionality.  I'm going to see if this can be done in my local fork.

**Comment by AustinWinstanley** [comment]
Yes, it was because of .Net Core. If you would like to put conditionals in there I will merge them.

**Comment by jblacker** [comment]
@AustinWinstanley I'm getting concurrency exceptions due to mismatching etags when using the embedded db.
